cv/classification/resnet50/ddp_inference.sh

```diff
@@ -0,0 +1,13 @@
+export PYTHONPATH=$PWD:$PYTHONPATH
+set -aux
+
+GPU_NUMS=8
+PORT=12346
+MODEL_ARCH="resnet50"  # "swin_tiny_patch4_window7_224"
+
+python3 -m oneflow.distributed.launch --nproc_per_node $GPU_NUMS --master_addr 127.0.0.1 --master_port $PORT  \
+        ../main.py --cfg ../configs/cnn_inference_settings.yaml --model_arch $MODEL_ARCH
```

Comment:
main.py
Assuming the user will run bash resnet50/infer.sh under the directory cv/classification.